hybrid-array/src/impls.rs

```diff
@@ -1,4 +1,22 @@
 use super::{Array, ArrayOps, ArraySize, IntoArray};
+use zeroize::{Zeroize, ZeroizeOnDrop};
+
+impl<T, U> Zeroize for Array<T, U>
+where
+    T: Zeroize,
+    U: ArraySize,
+{
+    fn zeroize(&mut self) {
+        self.0.as_mut().iter_mut().zeroize()
+    }
+}
+
+impl<T, U> ZeroizeOnDrop for Array<T, U>
+where
+    T: ZeroizeOnDrop,
+    U: ArraySize,
+{
+}
```

Comment:
A `zeroize` impl is good, but `Array` doesn't impl `Drop` and not every array needs to be zeroized on drop. Adding a `Drop` impl would also prevent moving the original array out of `Array`.

Reply:
I see. Array still is actually ZeroizeOnDrop tho right? Same reason `[Z; N]` is where `Z: ZeroizeOnDrop`

Reply:
Aah sorry, that makes sense